invisible_cities/reco/krmap_components.py

```diff
@@ -0,0 +1,360 @@
+import numpy          as np
+import pandas         as pd
+import scipy.stats    as stats
+import scipy.optimize as so
+
+from typing                               import Tuple
+from invisible_cities.core.core_functions import in_range, shift_to_bin_centers
+
+
+class KrMap():
```

Comment:
I'll look into this now! I'll tell you if I struggle with anything.

Reply:
I have a question here, maybe due to my limited knowledge, but why are we creating a new `KrMap` class instead of using the `ASectorMap` class that already exists and was used in ICAROS?

Reply:
Ok, I see, `ASectorMap` is already defined in `reco/corrections.py` but as far as I can see it's never used in the codebase. We should decide if we want to expand that, or create a new dataclass (`KrMap`?) and delete the unused `ASectorMap`. If we want to keep `KrMap`, I believe it needs a `chi2` attribute as well.

Reply:
I haven't looked too much into the whole organization of the code, but I believe we are discarding all the data model form icaros. 

> If we want to keep KrMap, I believe it needs a chi2 attribute as well.

This can be a column of the dataframe help by `KrMap`